roles/matrix-synapse/vars/workers.yml

```diff
@@ -74,6 +74,19 @@ matrix_synapse_workers_generic_worker_endpoints:
   - ^/_matrix/client/(api/v1|r0|v3|unstable)/join/
   - ^/_matrix/client/(api/v1|r0|v3|unstable)/profile/
 
+  # Device requests
+  - ^/_matrix/client/(r0|v3|unstable)/sendToDevice/
+
+  # Account data requests
+  - ^/_matrix/client/(r0|v3|unstable)/.*/tags
+  - ^/_matrix/client/(r0|v3|unstable)/.*/account_data
```

Comment:
Does this mean that the account_data endpoints are currently processed by the main process?